api/src/services/users.js

```diff
@@ -464,7 +464,7 @@ module.exports = {
   createUser: data => {
     const missing = missingFields(data);
     if (missing.length > 0) {
-      return Promise.reject(error400('Missing required fields: ' + missing.join(', ')));
+      return Promise.reject(error400('Missing required fields: ','fields.required', { 'fields': missing.join(', ')} ));
```

Comment:
We should keep the fields list in the static message as well, not just as a translation param.